core/common/blob.hpp

```diff
@@ -60,44 +68,35 @@ namespace kagome::common {
      * @return result containing Blob object if string has proper size,
      * otherwise result contains string with error message
```

Comment:
Not a string, but an error. Also please fix it everywhere result was refactored

Reply:
Ouch. I was fixing it while refactoring the return type, but seems like I missed some. Please tell me if you notice more.